python/pyarrow/tests/strategies.py

```diff
@@ -266,10 +277,12 @@ def arrays(draw, type, size=None, nullable=True):
         min_datetime = datetime.datetime.fromtimestamp(min_int64 // 10**9)
         max_datetime = datetime.datetime.fromtimestamp(max_int64 // 10**9)
         try:
-            offset_hours = int(ty.tz)
-            tz = pytz.FixedOffset(offset_hours * 60)
+            offset = ty.tz.split(":")
+            offset_hours = int(offset[0])
+            offset_min = int(offset[1])
+            tz = datetime.timedelta(hours=offset_hours, minutes=offset_min)
```

Comment:
Yes, that is correct, this isn't actually tested. But it would be good to update the `timezone` strategy to include fixed offsets. Will make a JIRA for it and leave this part of the code as is.

Reply:
https://issues.apache.org/jira/browse/ARROW-15887